Tests/PlotTests/HTMLTests.swift

```diff
@@ -189,6 +189,11 @@ final class HTMLTests: XCTestCase {
         let html = HTML(.body(.class("a"), .class("b")))
         assertEqualHTMLContent(html, #"<body class="b"></body>"#)
     }
+    
+    func testAppendingBodyCSSClass() {
+        let html = HTML(.body(Node.class("a").append("b")))
```

Comment:
I get what you were going for with this API design, but the problem is that it sort of "breaks" the DSL, by forcing you to refer to `Node` explicitly. Ideally, the end user should never have to worry much about the `Node` type, and instead be able to just add elements and attributes purely using dot-syntax. So I think it'd be better to enable this to be expressed as `.body(.append(.class("a")))` instead.